### PR TITLE
feat: Add Quickstart

### PR DESCRIPTION
**Pull request by dianacarvalho1**
Also update tycho url to point to prod

### CODE REVIEW

File: examples/quickstart/main.rs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            .await
+            .build()
+            .await
+            .expect("Failed building protocol stream");
+
+        while let Some(msg) = protocol_stream.next().await {
+            tick_tx
+                .send(msg.unwrap())
+                .await
+                .expect("Sending tick failed!");
```
Comment (kayibal) [inline examples/quickstart/main.rs:64]: Why not just do the printing here? I think having the channels makes the example unnecessarily complex.

File: examples/quickstart/main.rs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        }
+        anyhow::Result::Ok(())
+    });
+
+    // Consumer task: Process messages and perform calculations
+    let consumer_handle = task::spawn(async move {
+        let mut pairs: HashMap<String, Vec<Token>> = HashMap::new();
+        while let Some(message) = tick_rx.recv().await {
+            println!(
```
Comment (kayibal) [inline examples/quickstart/main.rs:73]: I'd put this in a function called: `print_prices(...)`

File: examples/quickstart/main.rs [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        .map(|result| {
+                            println!(
+                                "Amount out for trading 1 {:?} -> {:?}: {:?} (takes {:?} gas)",
+                                &tokens[0].symbol, &tokens[1].symbol, result.amount, result.gas
+                            )
+                        })
+                        .map_err(|e| {
+                            eprintln!("Error calculating amount out for Pool {:?}: {:?}", id, e)
+                        })
+                        .ok();
```
Comment (zizou0x) [inline examples/quickstart/main.rs:115]: Not needed I think? This result isn't used

**Review by zizou0x — APPROVED** [review]

File: examples/quickstart/main.rs (hunk 3 above)
Comment (dianacarvalho1) [reply examples/quickstart/main.rs:115]: if I don't put it, I need to add a `let _ =` in the beginning. I prefer the `ok()`

**Review by kayibal — APPROVED** [review]

**Pull request merged**